cypress/component/ProfileHeader.cy.jsx

```diff
@@ -9,13 +12,87 @@ describe("Render ProfileHeader Componenet ", () => {
     email: "email.com",
     picturePath: user1,
     following: ["testuser02", "testuser03"],
-    followers: ["testuser02", "testuser03", "testuser03"],
+    followers: ["testuser02", "testuser03"],
     recipes: ["testrecipe01", "testrecipe02"],
-    reviews: ["testreview01", "testreview02"],
+    reviews: [
+      {
+        recipeId: {
```

Comment:
Why is recipeId not an ID?

Reply:
well it is, but mongoose will populate into that object